pkgs/top-level/all-packages.nix

```diff
@@ -4540,6 +4542,7 @@ with pkgs;
 
   vowpal-wabbit = callPackage ../applications/science/machine-learning/vowpal-wabbit {
     boost = boost17x;
+    fmt = fmt_8;
```

Comment:
maybe just `propagatedBuilldInputs = [ fmt_8 ]` in `spdlog_1` instead of populating `fmt_8` so many times ?